### DISCUSSION TITLE
Is there a note taking system?

### DISCUSSION THREAD

**Discussion by MMesch** [ad-si/TaskLite#127] [Q&A] [answered]
Does tasklite plan to have a note taking system? In taskwarrior this is available through addons like _hasknote_.

**Comment by ad-si** [comment, marked as answer]
TaskLite itself is already kind of a note taking system. Each task itself can be considered a note: E.g. if you run `tl enter` it will create a new empty task and show it in your editor as a markdown document with frontmatter. The frontmatter contains the task data, and the markdown document body is the body of the task. If you want to better differentiate between tasks and notes you could simply give a `note` tag to every note.

Furthermore, if you want to add notes to existing tasks, you can simply use the `note` subcommand. 
E.g. `tl note 'Ask John about the details' 01kep2hnhtpff3mr0d2ztpwms0` (However, this feature will probably be renamed to `tl comment` as that seems more appropriate)

What features are you missing to consider TaskLite as a note taking system?
